src/Middleware/OutputCaching/src/OutputCacheServiceCollectionExtensions.cs

```diff
@@ -50,6 +50,8 @@ public static IServiceCollection AddOutputCache(this IServiceCollection services
         ArgumentNullException.ThrowIfNull(services);
         ArgumentNullException.ThrowIfNull(configureOptions);
 
+        services.AddTransient<IConfigureOptions<OutputCacheOptions>, OutputCacheOptionsSetup>();
```

Comment:
Can we instead move `AddOutputCache` above `Configure` since `AddOutputCache` already calls this?